articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -50,6 +50,10 @@ If you already have a data labeling project and you want to use that data, you c
 ### Using pre-labeled training data
 If you have previously labeled data that you would like to use to train your model, you will first need to upload the images to the default Azure Blob Storage of your Azure ML Workspace and register it as a data asset. 
 
+Below script uploads the image data at path "./data/odFridgeObjects" on your local machine to Azure Blob Storage. Thereafter, it creates a new data asset with the name "fridge-items-images-object-detection" in your Azure ML Workspace. Datastore property of this newly created data asset is made to point to location in Azure Blob Storage where the image data was uploaded. 
+
+If there already exists a data asset with name "fridge-items-images-object-detection" in your Azure ML Workspace, then it'll update its version number and update its datastore property to point to new location in Azure Blob Storage where we uploaded the image data.
```

Comment:
update its version number and update its datastore property - >update its version number and datastore property?

Reply:
The later part gets continued as "update its datastore property to point to new location in Azure Blob Storage". If we do 
"update its version number and datastore property to point to new location in Azure Blob Storage". It'd imply version number would also point to new location in Azure blob storage ?

Above was my understanding. But, please let me know your perspective, I am more than happy to correct it